### DISCUSSION TITLE
İhtiyaç/Talep Karşılandı Bilgisinin Girilmesi

### DISCUSSION THREAD

**Discussion by canotur** [acikkaynak/deprem-yardim-projesi#115] [Ideas]
Selamlar,

Öneri:
Aşağıdaki linkte de belirttiğim üzere ihtiyacı/talebi karşılanan kişilerin bu bilgiyi girebileceği bir arayüz tasarlanabilir. Bunun için talep/ihtiyaç girişi aşamasında kullanıcılara afetharita.com adresinde gözüken ID değeri ile yanında bir şifre verilebilir. Hatta güvenlik için eğer yapılabiliyorsa bu bilgiler SMS olarak gönderilebilir. Aynı zamanda kişilerin ihtiyaç/talepleri giderildiğinde bu bilgiyi iletmeleri hakkında gerekli bilgilendirmeler yapılabilir.

AfetHarita.com adresine ihtiyacı/talebi karşılayan STK kuruluşlarının bu bilgiyi girebileceği bir yetkinlik eklenebilir. Siteye yetkilendirme mekanizması eklenebilir veya Sadece yetkili kişiler tarafından güncelleme yapılabilecek basit bir şekilde ayrı bir ekran da yapılabilir. 

Fayda:
Böylece ihtiyacı/talebi giderilmiş kişilerin haritada güncellenmesi daha hızlı ve etkili olacaktır.

https://www.linkedin.com/feed/update/urn:li:activity:7030100127629361152?commentUrn=urn%3Ali%3Acomment%3A%28activity%3A7030100127629361152%2C7032808103100227584%29&dashCommentUrn=urn%3Ali%3Afsd_comment%3A%287032808103100227584%2Curn%3Ali%3Aactivity%3A7030100127629361152%29

İyi çalışmalar dilerim.

**Comment by agitrubard** [comment]
@canotur Selamlar,
Bu konuları da içeren bir proje için discussion oluşturmuştuk. İsterseniz oraya dahil edebiliriz, bu sayede fikirler birbirini ezmemiş olur ve fikir birliği ile birlikte ortaya çok iyi şeyler çıkabilir.

https://github.com/acikkaynak/afet-org/discussions/35

Bu konu üzerine hem fikir olursak Discord üzerinden bana ulaşabilirsiniz;
agitrubard#7048
